R/combined.R

```diff
@@ -286,17 +293,40 @@ combined_aggregate_data <- function(data) {
 }
 
 
-combined_aggregate_rt <- function(rt, samples) {
+combined_aggregate_severity <- function(severity, samples) {
+
+  severity <- switch_levels(severity)
+  hosp_names <- grep("hfr", names(severity), value = TRUE)
+  inf_names <- grep("hfr", names(severity), invert = TRUE, value = TRUE)
+
+  admissions <- switch_levels(severity[c("date", "step", hosp_names)])
+  infections <- switch_levels(severity[inf_names])
+
+
+  admissions <- combined_aggregate_rt(admissions, samples,
+                                      weight = "admitted_inc")
```

Comment:
`admitted_inc` is just confirmed admissions - really you'd want `admitted_inc + diagnoses_inc`

Reply:
This is now taken care of in the refactored function